dts/bindings/gpio/adi,max22190-gpio.yaml

```diff
@@ -0,0 +1,116 @@
+  mean WB functionality and fiters could be set.Reffering to:
+  filter-wbes = <CH0 CH1 CH2 ... CH7 >  for wire break
+  To ebale wire break set 1 else 0. Same principal is followed for:
+  filter-fbps and filter-dealys.
+  Sample binding
+  &sdp_spi {
+    status = "okay";
+    pinctrl-names = "default";
+    max22190_gpio0: max22190_gpio@0 {
+      compatible = "adi,max22190-gpio";
+      reg = <0>;
+
+      spi-max-frequency = <1000000>;
+      status = "okay";
+
+      gpio-controller;
+      #gpio-cells = <2>;
+      ngpios = <8>;
+      status = "okay";
+
+      max22190-mode = <1>; // modes range from 0-4
+
+      drdy-gpios = <&gpioj 12 GPIO_ACTIVE_LOW>; /* SDP-GPIO5 - PMOD-PIN8 */
+      fault-gpios = <&gpioc 11 GPIO_ACTIVE_LOW>; /* SDP-SERIAL_INT - PMOD-PIN7 */
+      latch-gpios = <&gpioj 13 GPIO_ACTIVE_LOW>; /* SDP-GPIO6 - PMOD-PIN9 */
+
+      filter-wbes = <1 0 0 0  0 0 0 0>; // wirebreak WBEx
+      filter-fbps = <0 0 0 0  0 0 0 0>; // programmable filter INx
+      filter-delays = <50 100 400 800 1600 3200 12800 20000>; // 1000 us == 1ms
+    };
+  };
+
+compatible: "adi,max22190-gpio"
+
+properties:
+  "#gpio-cells":
+    const: 2
+  ngpios:
+    type: int
+    required: true
+    const: 8
+    description: Number of gpios supported
+  drdy-gpios:
+    description: Ready pin which show when chip is ready
+    type: phandle-array
+  fault-gpios:
+    description: |
+      Fault pin indicates when there is Fault state in either FAULT1 or FAULT2
+      bothe of which are cleaned on read once problem is not persistent
+    type: phandle-array
+  latch-gpios:
+    description: |
+      Latch the data so it could be read (partially duplicate CS)
+    type: phandle-array
+  max22190-mode:
+    type: int
+    default: 1
+    required: true
+    enum:
+      - 0
+      - 1
+      - 2
+      - 3
+    description: |
+      Default mode set to 1, because in MAX22190PMB devkit this is hardwired
+      mode set by manifacturer.
+      max22190 mode is configured from M0 and M1 pins with
+      pull up or down resistors.
+      MODE| M1| M0| FRAME  | CRC | DAISY CHAIN
+      ----+---+---+--------+-----+------------
+        0 | 0 | 0 | 24-bit | yes | no
+        1 | 0 | 1 | 16-bit | no  | no
+        2 | 1 | 0 | 24-bit | yes | yes
+        3 | 1 | 1 | 16-bit | no  | yes
+  filter-wbes:
+    type: array
+    default: [1, 1, 1, 1, 1, 1, 1, 1]
+    description: |
+      Wire break functionality is enabled by default on all channels.
+      WBE bit in all Filter registers stand for wire break enable on each
+      channel, so to enable WB functionality set 1.
+      If WB on specific channel is disabled , FAULT will not be rised in case
+      wire is cut.
+      - 1 wire break enable
+      - 0 wire break disable
+      channels indentation start from CH0...CH7
+  filter-fbps:
+    type: array
+    default: [1, 1, 1, 1, 1, 1, 1, 1]
+    description: |
+      By default all channels are set to bypass mode from documentation.
```

Comment:
word this more like "The default value corresponds to the default value of the hardware"